src/stockwatch/analysis.py

```diff
@@ -19,6 +21,27 @@ def _create_figure() -> go.Figure:
     )
 
 
+def _add_plot_returns_line(
+    fig: go.Figure,
+    dates: list[datetime.date],
+    data: list[float],
+    title: str,
+    color: str,
+) -> None:
+    fig.add_trace(
```

Comment:
Test for existence of data[0]?

Reply:
not sure why? Previously we only checked if data[0] was not zero to avoid the unnecessary dashed lines. That is no longer really needed. The `data[0]` is safe because it's in the list comprehension, so only will be accessed if data has at least one element.